src/Api/Controllers/CiphersController.cs

```diff
@@ -224,8 +224,19 @@ public async Task<CipherMiniResponseModel> PutAdmin(string id, [FromBody] Cipher
                 throw new NotFoundException();
             }
 
-            var ciphers = await _cipherRepository.GetManyByUserIdAsync(userId, true);
-            var orgCiphers = ciphers.Where(c => c.OrganizationId == orgIdGuid);
+            IEnumerable<Cipher> orgCiphers;
+            if (await _currentContext.OrganizationOwner(orgIdGuid))
+            {
+                // User may be a Provider for the organization, in which case GetManyByUserIdAsync won't return any results
+                // But they have access to all organization ciphers, so we can safely get by orgId instead
```

Comment:
Nitpick One comment is before the IEnumerable, and another inside the if statement.